x-pack/plugins/ml/public/application/components/ml_inference/components/review_and_create_pipeline.tsx

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useMemo } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import {
+  EuiAccordion,
+  EuiCallOut,
+  EuiCodeBlock,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+  htmlIdGenerator,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { IngestPipeline } from '@elastic/elasticsearch/lib/api/types';
+import { useMlKibana } from '../../../contexts/kibana';
+
+interface Props {
+  inferencePipeline: IngestPipeline;
+  modelType?: string;
+  pipelineName: string;
+  pipelineCreated: boolean;
+  pipelineError?: string;
+}
+
+export const ReviewPipeline: FC<Props> = ({
+  inferencePipeline,
+  modelType,
+  pipelineName,
+  pipelineCreated,
+  pipelineError,
+}) => {
+  const {
+    services: {
+      docLinks: { links },
+    },
+  } = useMlKibana();
+
+  const inferenceProcessorLink =
+    modelType === 'regression'
+      ? links.ingest.inferenceRegression
+      : links.ingest.inferenceClassification;
+
+  const accordionId = useMemo(() => htmlIdGenerator()(), []);
+
+  const configCodeBlock = useMemo(
+    () => (
+      <EuiCodeBlock language="json" isCopyable overflowHeight="400px">
+        {JSON.stringify(inferencePipeline ?? {}, null, 2)}
+      </EuiCodeBlock>
+    ),
+    [inferencePipeline]
+  );
+
+  return (
+    <>
+      <EuiFlexGroup direction="column" gutterSize="s">
+        <EuiFlexItem grow={3}>
+          {pipelineCreated === false ? (
+            <EuiTitle size="s">
+              <h4>
+                {i18n.translate(
+                  'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.review.title',
+                  {
+                    defaultMessage: "Review the pipeline configuration for '{pipelineName}'",
+                    values: { pipelineName },
+                  }
+                )}
+              </h4>
+            </EuiTitle>
+          ) : null}
+          <>
+            <EuiSpacer size="s" />
+            {pipelineCreated === true && pipelineError === undefined ? (
```

Comment:
I think we need a link to the Ingest pipelines page in Stack Management after the pipeline has been created.